src/main/java/io/github/ImpactDevelopment/installer/versions/VanillaProfiles.java

```diff
@@ -15,7 +15,7 @@
  * along with Impact Installer.  If not, see <https://www.gnu.org/licenses/>.
  */
 
-package io.github.ImpactDevelopment.installer.profiles;
```

Comment:
I've been trying to think of a better name for the `versions` package tbh.

Origionally when I added `versions` and `profiles` I envisioned having multiple profile and version implementations for different launchers.

We hadn't added releases then, but now that we have, `versions` is probably a confusing name.

I'm not sure if profiles and versions should be in the same package or not either.

Maybe `targets`, howabout renaming `profiles` to `launchers` and putting all launcher stuff (including directory finding and `install()`) in there?

Reply:
Hm, so `targets` would have... `InstallationMode` and `InstallationModeOptions`.... and `launchers` would have `Vanilla`, `VanillaProfiles`, and `Forge`?

Reply:
Yeah I guess so, although maybe `launchers` isn't the best name for something that includes forge. Also what if we wan't to install forge on multimc or some other launcher? Technically our forge is still an installmode of vanilla launcher since it uses `.minecraft/mods`